### PR TITLE
Update docs to remove auto-tables

### PR DESCRIPTION
**Pull request by markstory**
Don't highlight autotables in the docs.

Refs cakephp/cakephp#15152

### CODE REVIEW

File: en/orm.rst [hunk 1]
```diff
@@ -25,56 +25,55 @@ database connections <database-configuration>`.
...
 
-    public function someMethod()
+    use Cake\ORM\Table;
+
+    class ArticlesTable extends Table
     {
-        $articles = $this->getTableLocator()->get('Articles');
+    }
+
+Then in a controller, or command we can have CakePHP create an instnace for us::
```
Comment (dereuromark) [inline en/orm.rst]:
```suggestion
Then in a controller, or command we can have CakePHP create an instance for us::
```

File: en/orm.rst [hunk 2]
```diff
@@ -25,56 +25,55 @@ database connections <database-configuration>`.
...
-Table classes use the CamelCased version of the table name with the ``Table``
-suffix as the class name. Once your class has been created you get a reference
-to it using the :php:class:`~Cake\\ORM\\Locator\\TableLocator` as before::
-
-    use Cake\ORM\Locator\LocatorAwareTrait;
+Within a static method you can use the :php:class:`~Cake\\Datasource\\FactoryLocator` 
+to get the table locator::
 
-    $articles = $this->getTableLocator()->get('Articles');
+    $articles = FactoryLocator::get('Table')->get('Articles');
```
Comment (dereuromark) [inline en/orm.rst]: This should be using TableRegistry::getTableLocator()-> instead IMO.
The magic string one is a bit weird for app level usage.

File: en/appendices/4-2-migration-guide.rst [hunk 3]
```diff
@@ -44,6 +44,10 @@ Database
 ORM
 ---
 
+- ``TableLocator::allowFallbackClass()`` was added. This method lets you
+  disable automatically generated fallback table classes. This feature is
+  currently an opt-in feature but will become a enabled by default in the
```
Comment (dereuromark) [inline en/appendices/4-2-migration-guide.rst]: I think it is the opposite: It is currently opt-out by default (enabled always by default), and will require opt-in in the future.

Comment (markstory) [reply en/appendices/4-2-migration-guide.rst]: I meant to say that disabling auto-tables is opt-in though.

File: en/orm.rst (hunk 2 above)
Comment (markstory) [reply en/orm.rst]: This example has been in the docs for a while.

File: en/appendices/4-2-migration-guide.rst [hunk 4]
```diff
@@ -44,6 +44,10 @@ Database
...
+  future.
```
Comment (othercorey) [inline en/appendices/4-2-migration-guide.rst]:
```suggestion
  disable automatically generated fallback table classes. Disabling is
  currently opt-in, but will become the default in the future.
```

File: en/orm.rst [hunk 5]
```diff
@@ -25,56 +25,55 @@ database connections <database-configuration>`.
...
-To get started you don't have to write any code. If you've followed the :ref:`CakePHP
-conventions for your database tables <model-and-database-conventions>`
-you can just start using the ORM. For example if we wanted to load some data 
-from our ``articles`` table we could do::
+To get started you don't have to write any code. If you've followed the
+:ref:`CakePHP conventions for your database tables
+<model-and-database-conventions>` you can just start using the ORM. For example
+if we wanted to load some data from our ``articles`` table we would start off
+creating our ``Articles`` table class. Add the following to
+**src/Model/Table/ArticlesTable.php** after the ``<?php`` opening tag::
```
Comment (othercorey) [inline en/orm.rst]:
```suggestion
creating our ``Articles`` table class. Create
**src/Model/Table/ArticlesTable.php** with the following code::
```

File: en/orm.rst [hunk 6]
```diff
@@ -25,56 +25,55 @@ database connections <database-configuration>`.
...
 
-    public function someMethod()
+    use Cake\ORM\Table;
+
+    class ArticlesTable extends Table
     {
-        $articles = $this->getTableLocator()->get('Articles');
+    }
+
+Then in a controller, or command we can have CakePHP create an instance for us::
```
Comment (othercorey) [inline en/orm.rst]:
```suggestion
Then in a controller or command we can have CakePHP create an instance for us::
```

File: en/orm.rst [hunk 7]
```diff
@@ -25,56 +25,55 @@ database connections <database-configuration>`.
...
-Note that we didn't have to create any code or wire any configuration up.
-The conventions in CakePHP allow us to skip some boilerplate code and allow the
-framework to insert base classes when your application has not created
-a concrete class. If we wanted to customize our ArticlesTable class adding some
-associations or defining some additional methods we would add the following to
-**src/Model/Table/ArticlesTable.php** after the ``<?php`` opening tag::
 
-    namespace App\Model\Table;
+In other contexts, you can use the ``LocatorAwareTrait`` to add methods that
+enable you to get ORM tables::
```
Comment (othercorey) [inline en/orm.rst]:
```suggestion
In other contexts, you can use the ``LocatorAwareTrait`` which add accessor methods for ORM tables::
```

File: en/orm.rst (hunk 2 above)
Comment (dereuromark) [reply en/orm.rst]: Yeah, i still think its wrong to leak that internal detail.

**Pull request merged**